ee/clickhouse/queries/funnels/funnel_time_to_convert.py

```diff
@@ -43,7 +43,7 @@ def get_query(self) -> str:
             bin_count_identifier = "bin_count"
             bin_count_expression = f"""
                 count() AS sample_count,
-                least(60, greatest(3, ceil(cbrt(sample_count)))) AS {bin_count_identifier},
+                ifNull(least(60, greatest(3, ceil(cbrt(sample_count)))), 0) AS {bin_count_identifier},
```

Comment:
Do we need both of the `ifNull` additions?

Is there a reason to not put this directly on the `count()`?